management_api_app/models/schemas/workspace.py

```diff
@@ -24,7 +24,7 @@ def get_sample_workspace(workspace_id: str, spec_workspace_id: str = "0001") ->
             "status": "not_deployed",
             "message": "This resource is not yet deployed"
         },
-        "isDeleted": False,
+        "deleted": False,
         "resourceType": "workspace",
```

Comment:
I know this is not in the changed code, but as you are changing adjacent code - we should replace workspace by the enum (so that we change it only in one place when we change it)